code/game/atoms.dm

```diff
@@ -517,6 +517,9 @@
 /atom/proc/cmag_act()
```

Comment:
doesn't the only place this is called pass a `mob/user` parameter? Why not use that instead of using `usr` in overrides of this proc?

Reply:
Fixed!